Makefile

```diff
@@ -327,6 +327,8 @@ endif
 
 check-style: golangci-lint plugin-checker vet ## Runs golangci against all packages
 
+check-style-ci: plugin-checker vet ## Runs plugin-checker and vet
```

Comment:
It's not very clear why there's a `check-style` and `check-style-ci`. I'd probably run `make plugin-checker` and `make vet` serially. 0/5.

Reply:
sounds reasonable